build-logic/dependency-modules/src/main/kotlin/gradlebuild.dependency-modules.gradle.kts

```diff
@@ -247,3 +249,17 @@ abstract class ReplaceCglibNodepWithCglibRule : ComponentMetadataRule {
         }
     }
 }
+
+// https://youtrack.jetbrains.com/issue/IDEA-261387
+abstract class UnifyTrove4jVersionRule : ComponentMetadataRule {
+    override fun execute(context: ComponentMetadataContext) {
+        context.details.allVariants {
+            withDependencies {
+                if (any { it.name == "trove4j" }) {
+                    removeAll { it.name == "trove4j" }
+                    add("org.jetbrains.intellij.deps:trove4j:trove4j:1.0.20181211")
```

Comment:
We manage the version via our platform. Why do we need to specify the version here again?